gapic-generator/lib/gapic/presenters/service_presenter.rb

```diff
@@ -61,6 +61,10 @@ def package
         PackagePresenter.new @gem_presenter, @api, @service.parent.package
       end
 
+      def special_compute_behavior?
+        address[0] == "google" && address[1] == "cloud" && address[2] == "compute" && address[3] == "v1"
```

Comment:
Maybe we want this to be not empty instead of a specific version? If someone generates code for beta do shouldn't they get pagination?

Reply:
sgtm